overlord/snapstate/snapstate.go

```diff
@@ -3803,5 +3875,14 @@ func MockEnforceSingleRebootForBaseKernelGadget(val bool) (restore func()) {
 	return func() {
 		enforcedSingleRebootForGadgetKernelBase = old
 	}
+}
 
+func MockEnforceValidationSets(f func(*state.State, map[string]*asserts.ValidationSet, map[string]int, []*snapasserts.InstalledSnap, map[string]bool, int) error) func() {
```

Comment:
We don't do anything like this for any of the other global hooks, we just have code to set them/unset them in the tests

Reply:
We do the same for `EnforcedValidationSets`. Should I remove both?

Reply:
yes, I don't see anything else like MockEnforcedValidationSets, it is used in hookstate and snapshotstate tests afaict